model/job_queue/v1/queue_resource.model

```diff
@@ -27,10 +27,10 @@ resource Queue {
         out Kind String
         out HREF String
         in out Arguments String
-        out Attempts Integer
-        out AbandonedAt Date
+        in out Attempts Integer
+        in out AbandonedAt Date
```

Comment:
Same. Without additional documentation I would probably assume that this means that the job will be automatically abandoned at the date that the user specifies here. Is that correct?